src/coprocessor/mod.rs

```diff
@@ -22,7 +22,10 @@ use crate::store::Store;
 /// - A trait [`crate::coprocessor::Coprocessor`], which defines the methods and behavior for all coprocessors.
 /// - An enum such as [`crate::eval::lang::Coproc`], which "closes" the hierarchy of possible coprocessor
 ///   implementations we want to instantiate at a particular point in the code.
-pub trait Coprocessor<F: LurkField>: Clone + Debug + Sync + CoCircuit<F> {
+pub trait Coprocessor<F: LurkField>: Clone + Debug + Sync + Send + CoCircuit<F> {
+    fn new() -> Self {
+        unimplemented!();
+    }
```

Comment:
Nit: this is unusual, what am I missing?

Reply:
Nothing, removed in follow-up — just a development artifact.